### PR TITLE
Version bump to 1.1.0. Update Changelog. Update package metadata.

### PR DESCRIPTION
**Pull request by WeatherGod**
Don't merge yet. Staging the changes needed for a version 1.1.0 release of Basemap.
Update the package metadata to include myself as maintainer, updated the changelog, and did a version bump.

I should double-check that the license information is correct.

### CODE REVIEW

File: doc/users/installing.rst [hunk 1]
```diff
@@ -21,28 +21,27 @@ Python 2.4 (or later, including Python 3)
 numpy 1.2.1 (or later)
     array support for python (`download <http://sourceforge.net/project/showfiles.php?group_id=1369&package_id=175103>`__)
 
-**Required libraries that ship with basemap**
+`PROJ4 <http://trac.osgeo.org/proj/>`__ Cartographic Projections Library.
```
Comment (QuLogic) [inline doc/users/installing.rst]: https

File: doc/users/intro.rst [hunk 2]
```diff
@@ -21,14 +21,30 @@ datasets (from `Generic Mapping Tools <http://gmt.soest.hawaii.edu/>`_)
...
-contributed new features.  
+contributed new features.
+
+Cartopy, New Management, and EoL Announcement
+=============================================
+
+Starting in 2016, Basemap came under new management. The
+`Cartopy project <http://scitools.org.uk/cartopy/>`_ will replace
+Basemap, but it hasn't yet implemented all of Basemap's features.
+All new software development should try to use CartoPy whenever possible,
```
Comment (QuLogic) [inline doc/users/intro.rst]: An errant CartoPy.

File: doc/users/installing.rst [hunk 3]
```diff
@@ -21,28 +21,27 @@ Python 2.4 (or later, including Python 3)
 numpy 1.2.1 (or later)
     array support for python (`download <http://sourceforge.net/project/showfiles.php?group_id=1369&package_id=175103>`__)
 
-**Required libraries that ship with basemap**
+`PROJ4 <https://trac.osgeo.org/proj/>`__ Cartographic Projections Library.
+
+**Required library that ship with basemap**
```
Comment (efiring) [inline doc/users/installing.rst]: ship -> ships

File: doc/users/intro.rst [hunk 4]
```diff
@@ -21,14 +21,30 @@ datasets (from `Generic Mapping Tools <http://gmt.soest.hawaii.edu/>`_)
 are provided, along with methods for plotting them. The `GEOS library 
 <http://geos.refractions.net>`_ is used internally to clip the coastline and polticial boundary features to the desired map projection region.
 
-Basemap provides facilities for reading `shapefiles
-<http://en.wikipedia.org/wiki/Shapefile>`_.
-
 Basemap is geared toward the needs of earth scientists, particular
```
Comment (efiring) [inline doc/users/intro.rst]: particular -> particularly (or "in particular")

File: doc/users/intro.rst [hunk 5]
```diff
@@ -21,14 +21,30 @@ datasets (from `Generic Mapping Tools <http://gmt.soest.hawaii.edu/>`_)
...
+`Cartopy project <http://scitools.org.uk/cartopy/>`_ will replace
+Basemap, but it hasn't yet implemented all of Basemap's features.
+All new software development should try to use Cartopy whenever possible,
+and existing software should start the process of switching over to use
+Cartopy. All maintenance and development efforts should be focused on
+Cartopy.
+
+Ben Root has volunteered to take over maintenance of Basemap until 2020.
+Pull requests will be reviewed, and bugs will be fixed. Also, the
+maintenance will ensure compatibility with packages like NumPy and matplotlib.
```
Comment (efiring) [inline doc/users/intro.rst]: Capitalize Matplotlib.

**Comment by WeatherGod** [comment]
going to try and figure everything out for release tomorrow

**Pull request merged**